packages/office-ui-fabric-react/src/components/DatePicker/DatePicker.base.tsx

```diff
@@ -184,8 +196,7 @@ export class DatePickerBase extends BaseComponent<IDatePickerProps, IDatePickerS
               onClick: this._onIconClick,
               className: css(
```

Comment:
This concatination logic should be in the styles.ts file.  There `css()` function should no longer be used.